src/images/all-of-us-logo.svg

```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It looks like SVGs are pretty widely supported these days, nice. http://caniuse.com/#feat=svg Do you know if there's a way we can find expected browser/device usage? I wonder if it's very different from in general on the web. Also, are there any contractual constraints on browser support (hopefully not)?

I don't know if we should care about file size much, or whether a compressed/crushed PNG would be smaller than this (especially if we have to produce multiple sizes for different devices).